run_scripts/convergence_function.py

```diff
@@ -0,0 +1,174 @@
+import argparse
```

Comment:
@zhangshixuan1987, could these 4 .py scripts please be put in a subfolder of run_scripts called "convergence_run"?

Reply:
@vlarson : Sure, I will modify it following your suggestion

Reply:
@vlarson : I have pushed the changes to this pull request. Please check if you can see the changes on your side.